driver/src/contracts/stablex_contract_auction_element.rs

```diff
@@ -0,0 +1,224 @@
+use byteorder::{BigEndian, ByteOrder};
+
+use std::collections::HashMap;
+
+use web3::types::{H160, U256};
+
+use dfusion_core::models::{BatchInformation, Order};
+
+#[derive(Debug, Default, PartialEq)]
```

Comment:
Why do we need Default?

Reply:
As we want to provide Default for AuctionElement

Reply:
Yes, needed for `AuctionElement::default()`